test/test_datasources.py

```diff
@@ -154,157 +137,193 @@ def test_find_agg_fail(self):
     assert timestamps.shape == (32,)
 
 
-def test_get_timestamps_last_timestep(netcdf_groundwater_ds):
-    timestamps = netcdf_groundwater_ds.get_timestamps()
+def test_get_timestamps_last_timestep(threedi_result):
+    timestamps = threedi_result.get_timestamps()
     assert timestamps[-1] == 1863.5643704609731
 
 
-def test_get_timestamps_with_agg_parameter_q_cum(netcdf_groundwater_ds):
-    timestamps_q_cum = netcdf_groundwater_ds.get_timestamps(parameter="q_cum")
+def test_get_timestamps_with_agg_parameter_q_cum(threedi_result):
+    timestamps_q_cum = threedi_result.get_timestamps(parameter="q_cum")
     assert timestamps_q_cum.shape == (7,)
     assert timestamps_q_cum[-1] == 1801.2566835460611
 
 
-def test_get_timestamps_with_agg_parameter_vol_current(netcdf_groundwater_ds):
-    timestamps_vol_current = netcdf_groundwater_ds.get_timestamps(
+def test_get_timestamps_with_agg_parameter_vol_current(threedi_result):
+    timestamps_vol_current = threedi_result.get_timestamps(
         parameter="vol_current"
     )
     assert timestamps_vol_current.shape == (7,)
     assert timestamps_vol_current[-1] == 1801.2566835460611
 
 
-def test_get_gridadmin(netcdf_groundwater_ds):
-    ga = netcdf_groundwater_ds.get_gridadmin(variable=None)
+def test_get_gridadmin(threedi_result):
+    ga = threedi_result.get_gridadmin(variable=None)
     assert isinstance(ga, gridresultadmin.GridH5Admin)
 
 
-def test_get_gridadmin_result_var(netcdf_groundwater_ds):
-    ga = netcdf_groundwater_ds.get_gridadmin(variable="s1")
+def test_get_gridadmin_result_var(threedi_result):
+    ga = threedi_result.get_gridadmin(variable="s1")
     assert isinstance(ga, gridresultadmin.GridH5ResultAdmin)
 
 
-def test_get_gridadmin_agg_result_var(netcdf_groundwater_ds):
-    ga = netcdf_groundwater_ds.get_gridadmin(variable="q_cum")
+def test_get_gridadmin_agg_result_var(threedi_result):
+    ga = threedi_result.get_gridadmin(variable="q_cum")
     assert isinstance(ga, gridresultadmin.GridH5AggregateResultAdmin)
 
 
-def test_get_gridadmin_agg_result_var_not_available(netcdf_groundwater_ds):
+def test_get_gridadmin_agg_result_var_not_available(threedi_result):
     with pytest.raises(AttributeError):
-        netcdf_groundwater_ds.get_gridadmin(variable="u1_max")
+        threedi_result.get_gridadmin(variable="u1_max")
 
 
-def test_get_gridadmin_unknown_var(netcdf_groundwater_ds):
+def test_get_gridadmin_unknown_var(threedi_result):
     with pytest.raises(AttributeError):
-        netcdf_groundwater_ds.get_gridadmin(variable="unknown")
+        threedi_result.get_gridadmin(variable="unknown")
 
 
-def test_get_timeseries(netcdf_groundwater_ds):
-    ts = netcdf_groundwater_ds.get_timeseries("s1")
-    np.testing.assert_equal(ts[:, 0], netcdf_groundwater_ds.get_timestamps())
-    assert ts.shape[1] == netcdf_groundwater_ds.get_gridadmin("s1").nodes.count + 1
+def test_get_timeseries(threedi_result):
+    ts = threedi_result.get_timeseries("s1")
```

Comment:
`timeseries`? Daar zit namelijk in dat het ook wel eens meerdere timeseries kunnen zijn.
`ts` kan als enkelvoud `timeserie` gelezen worden en daardoor voor foutjes zorgen.